thinkhazard/static/less/common.less

```diff
@@ -37,6 +37,13 @@ ul.horizontal li {
 .navbar {
   border: 0;
   .box-shadow(0px 2px 10px -1px rgba(0, 0, 0, 0.33));
+  li:last-child a{
+      padding-top: 0px;
+      padding-bottom: 0px;
+      margin-top: 29px;
+      margin-bottom: 29px;
```

Comment:
I changed the padding because I added the separator as a border. But the border has the same height as the text + the padding which makes the border too high (compared to what is on the mokup).
![separator](https://cloud.githubusercontent.com/assets/1460598/8497393/299aab1a-217d-11e5-912c-1f3cd898a32a.png)
If the padding is 0 and the margin to the height of the padding this fixes the problem. Perhaps there is a better way to do this?